redbox/storage/elasticsearch.py

```diff
@@ -210,3 +232,21 @@ def get_file_status(self, file_uuid: UUID, user_uuid: UUID) -> FileStatus:
             chunk_statuses=chunk_statuses,
             processing_status=ProcessingStatusEnum.complete if is_complete else ProcessingStatusEnum.embedding,
         )
+
+
+def hit_to_chunk(hit: Dict[str, Any]) -> Chunk:
```

Comment:
nice. this "inclusive" approach looks like the best way to start this major refactor.